database/repositories/profile_pictures.php

```diff
@@ -2,10 +2,9 @@
 global $db_host, $db_username, $db_password, $db_database, $con;
 require_once(__DIR__ . '/../../secrets.settings.php');
 
-function get_user_pfp($user) // base64
+function get_user_pfp(string $user): string|null // base64
```

Comment:
I would rename the function to get_user_pfp_by_id and change the parameter to $user_id